docs/pages/reference/cli.mdx

```diff
@@ -2034,3 +2028,221 @@ $ tsh ls --search=staging,mac
+
+```code
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --bot-user=teleport \
+    --reader-user=jenkins
+```
+
+
+### tbot db
+
+Connects to databases using native clients and queries database information. This is best used for testing and validation purposes;
+most users will likely prefer to connect their own databases to a local proxy using `tbot proxy db`.
+
+Note that tsh must be installed to make use of this command.
+
+#### Flags
+
+| Flag                | Description                                                                                              |
+|---------------------|----------------------------------------------------------------------------------------------------------|
+| `-d/--debug`        | Enable verbose logging to stderr.                                                                        |
+| `-c/--config`       | Path to a configuration file.                                                                            |
+| `--destination-dir` | Path to the Machine ID destination dir that should be used for authentication. Required.                 |
+| `--proxy`           | The `host:port` of the Teleport Proxy Service to use to access resources. Required.                      |
+| `--cluster`         | The name of the cluster on which resources should be accessed. Extracted from the bot identity if unset. |
+
+All other flags and arguments are passed directly to `tsh db ...`, along
+with authentication parameters to use the Machine ID identity to skip `tsh`'s
+login steps.
+
+Note that certain CLI parameters, for example `--help`, may be captured by
+`tbot` even if intended to be passed to the wrapped `tsh`. A `--` argument can
+be used to ensure all following arguments are passed to `tsh` and ignored by
+`tbot`.
+
+Additionally, be aware of the following limitations of `tbot db`:
+ - `tbot db connect` requires a `tbot db login` for certain database types,
+   like MySQL, so that additional connection parameters can be written to a
+   local configuration file.
+ - `tbot db env` is not fully supported.
+
+### tbot proxy
+
+Allows for access to teleport resources via a proxy using various protocols. Requires running a local ALPN/SNI proxy to access the resources if using TLS.
+The `tbot proxy` command acts as a wrapper for `tsh proxy` to provide local proxy functionality for various protocols.
+
+Note that tsh must be installed to make use of this command.
+
+#### Flags
+
+| Flag                | Description                                                                                              |
+|---------------------|----------------------------------------------------------------------------------------------------------|
+| `-d/--debug`        | Enable verbose logging to stderr.                                                                        |
+| `-c/--config`       | Path to a configuration file.                                                                            |
```

Comment:
Is this required? If not, what's the default?

Reply:
I'm still not sure what the default value for this is. Also, is this a `tbot` configuration profile or a `tsh` configuration file?